tests/Hilke.KineticConvolution.Tests/DirectionTests.cs

```diff
@@ -81,5 +84,69 @@ public void When_DirectionRange_Is_The_Complete_Circle_Then_All_Directions_Shoul
             westBelongsToClockwiseRange.Should().BeTrue();
             westBelongsToCounterClockwiseRange.Should().BeTrue();
         }
+
+        private static IEnumerable<TestCaseData> DirectionEqualityTestCaseSource()
+        {
+            var factory = new ConvolutionFactory();
```

Comment:
You could put that in the OneTimeSetup

Reply:
It's a bit tricky, because the `OneTimeSetUp` is called after the `TestCaseSource` are evaluated, meaning that `_factory` is `null` when the test case source is evaluated. The workaround is to statically initialize the `_factory` member.

Reply:
yeah, not worth it, leave it like that